docs/examples/userguide/parallelism/memoryview_sum.pyx

```diff
@@ -0,0 +1,9 @@
+from cython.parallel import prange
+
+
+
+def func(double[:] x, double alpha):
```

Comment:
This doesn't quite look like it'll line up with the .py file

Reply:
fixed.